app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java

```diff
@@ -196,7 +195,34 @@ public void onInput(@NonNull MaterialDialog dialog, CharSequence input) {
                         } else {
                             mFilename = input.toString();
 
-                            new CreatingPdf().execute();
+                            mPath = Environment.getExternalStorageDirectory().getAbsolutePath() +
+                                    HomeFragment.this.getString(R.string.pdf_dir);
+
+                            mPath = mPath + mFilename + HomeFragment.this.getString(R.string.pdf_ext);
+
+                            File tempFile = new File(mPath);
+                            if (tempFile.exists()) {
+                                new MaterialDialog.Builder(mActivity)
```

Comment:
Move this code (creating the dialog) to a new function. This function is becoming too complex.